pinot-query-runtime/src/test/resources/queries/QueryHints.json

```diff
@@ -321,6 +321,10 @@
         "description": "aggregate with skip intermediate stage hint (via hint option is_partitioned_by_group_by_keys)",
         "sql": "SELECT /*+ aggOptions(is_partitioned_by_group_by_keys='true') */ {tbl1}.num, COUNT(*), SUM({tbl1}.val), SUM({tbl1}.num), COUNT(DISTINCT {tbl1}.val) FROM {tbl1} WHERE {tbl1}.val >= 0 AND {tbl1}.name != 'a' GROUP BY {tbl1}.num"
       },
```

Comment:
Is there a test asserting that trimming affects results ?

Reply:
I can probably make one, but that will cause wrong (and inconsistent) result, so not sure how to test it. I've manually tested it

Reply:
I already added some tests in https://github.com/apache/pinot/pull/14727.
It was tricky because in many cases trimming makes result unstable.